spec/rails_spec_helper.rb

```diff
@@ -10,6 +10,23 @@ def testing_ruby_2?
   RUBY_VERSION.split('.')[0].to_i == 2
 end
 
+# Rails5 doesn't work with Ruby 3.x, Rails 7 doesn't work with Ruby < 2.7.
+def default_rails_versions
+  if testing_ruby_2?
+    if Gem::Requirement.create('>= 2.7') =~ Gem::Version.new(RUBY_VERSION)
+      [ 5, 6, 7 ]
+    else
+      [ 5, 6 ]
+    end
+  else
+    [ 6, 7 ]
+  end
+end
+
+def rails_versions
+  Array(ENV['RAILS_VERSIONS'] || default_rails_versions)
```

Comment:
This code was just moved around, but since you're already here anyway you might want to `split` the envar on some separator here, to allow specifying multiple versions. Otherwise the variable name is wrong ;)